tests/manage/pv_services/pvc_resize/test_pod_deletion_during_pvc_expansion.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import (
+    get_all_pods,
+    get_plugin_pods,
+    get_pod_node,
+    get_pod_logs,
+)
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    ManageTest,
+    tier4c,
+    bugzilla,
+    polarion_id,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier4c
+@bugzilla("2164617")
+@polarion_id("OCS-4877")
+@skipif_ocs_version("<4.13")
+class TestPodRespinDuringPvcExpansion(ManageTest):
+    """
+    Test to verify PVC expansion during app pod respins
+
+    """
+
+    def test_pod_respin_during_pvc_expansion(self, dc_pod_factory):
+        """
+        Verify PVC expansion during rbd-app pod respins
+
+        Test Steps:
+        * Create RBD Filesystem PVC
+        * Mount it to the application pod
+        * Expand the PVC
+        * Restart the pod multiple times
+        * Check csi-rbdplugin pod logs on the node where the app pod is running for error
+            'Internal desc = failed to get device for stagingtarget path'
+        * The expansion should be successful, and the app should be running
+
+        """
+        rbd_pod = dc_pod_factory(size=10)
```

Comment:
this test is valid for external mode too. Resource name changes in external mode, please check if it is handled in this fixture

Reply:
Looks like the dc_pod_factory handles it, resolving the comment.